app/app.html

```diff
@@ -30,5 +32,15 @@
         document.body.appendChild(script);
       }
     </script>
+
+    <script src="http://cdnjs.cloudflare.com/ajax/libs/react/15.3.2/react.min.js"></script>
+    <script src="http://cdnjs.cloudflare.com/ajax/libs/react/15.3.2/react-dom.min.js"></script>
+    <script src="http://unpkg.com/babel-core@5.8.38/browser.min.js"></script>
+
+    <script src="http://ajax.googleapis.com/ajax/libs/jquery/3.1.0/jquery.min.js"></script>
+    <script src="http://media.twiliocdn.com/sdk/js/client/v1.3/twilio.min.js"></script>
+    <script src="http://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/js/bootstrap.min.js"></script>
+
+    <script type="text/babel" src="dialer.jsx"></script>
```

Comment:
remove the react & react-dom since they are already in packages and will be bundled with webpack